orttraining/orttraining/test/python/orttraining_test_orttrainer_bert_toy_onnx.py

```diff
@@ -322,15 +333,18 @@ def testToyBERTModelLRScheduler(initial_lr, lr_scheduler, expected_learning_rate
     _test_helpers.assert_model_outputs(losses, expected_losses, rtol=1e-6)
     
 
-# TODO: Copy the expected values from legacy api
 # Dynamic Loss Scaler implemented implicitly
-@pytest.mark.parametrize("loss_scaler", [
-    (None),
-    (amp.DynamicLossScaler()),
-    (CustomLossScaler())
+@pytest.mark.parametrize("loss_scaler, expected_losses", [
+    (None, [10.98803424835205, 10.99240493774414, 11.090575218200684, 11.042827606201172, 10.988829612731934,
+        11.105679512023926, 10.981969833374023, 11.081724166870117, 10.997172355651855, 11.107305526733398]),
+    (amp.DynamicLossScaler(), [10.98803424835205, 10.99240493774414, 11.090575218200684, 11.042827606201172, 10.988829612731934,
+        11.105679512023926, 10.981969833374023, 11.08173656463623, 10.997119903564453, 11.107370376586914]),
+    (CustomLossScaler(), [10.98803424835205, 10.99240493774414, 11.090554237365723, 11.042823791503906, 10.98877239227295,
+        11.105667114257812, 10.981982231140137, 11.081768035888672, 10.997129440307617, 11.107320785522461])
 ])
-def testToyBERTModelMixedPrecisionLossScaler(loss_scaler):
+def testToyBERTModelMixedPrecisionLossScaler(loss_scaler, expected_losses):
```

Comment:
The `expected_losses` values were generated using a `seed`, which is not set in this test. Find out which seed was that and set it to both legacy and experimental runs